2.7/Dockerfile

```diff
@@ -6,12 +6,11 @@
 # specific reason to use a different one. This means January, April, July, or
 # October.
 
-# NOTE: This uses November due to an OpenSSL CVE fix. We will resume using quarterly versions of cimg-base in Q1
-FROM cimg/base:2022.11-20.04
+FROM cimg/base:2023.01
```

Comment:
As noted in #116, we need to use ubuntu 20.04.
https://github.com/CircleCI-Public/cimg-ruby/pull/116/files#diff-8d7a21b017921bb88eaf71656b7b5767203db16e8126fc1e5ad2a9ba0bc542f5R9